core/src/test/scala/integration/kafka/api/AdminClientIntegrationTest.scala

```diff
@@ -1819,6 +1819,123 @@ class AdminClientIntegrationTest extends IntegrationTestHarness with Logging {
       classOf[InvalidTopicException])
     client.close()
   }
+
+  @Test
+  def testDescribeConfigsForLog4jLogLevels(): Unit = {
+    createTopic(topic, numPartitions = 2, replicationFactor = brokerCount)
```

Comment:
why are you creating a topic?

Reply:
https://en.wikipedia.org/wiki/Copy-and-paste_programming